control_panel_api/authentication.py

```diff
@@ -29,8 +30,14 @@ def get_jwt(request):
 
 
 def get_jwks():
-    response = requests.get(settings.OIDC_WELL_KNOWN_URL)
-    return response.json()
+    jwks = cache.get(settings.OIDC_WELL_KNOWN_URL)
```

Comment:
I know this wasn't named in this commit, but `OIDC_WELL_KNOWN_URL` would be better named `OIDC_JWKS_URL` or something.

Reply:
Agreed. But I'm not going into the potential rabbit hole of renaming this. This meant to be a quick win :)